recipes/libbacktrace/meta.yaml

```diff
@@ -0,0 +1,31 @@
+{% set version = "1.0" %}
+
+package:
+    name: backtrace
+    version: {{ version }}
+
+source:
+    fn: 177940370e4a6b2509e92a0aaa9749184e64af43.zip
+    url: https://github.com/ianlancetaylor/libbacktrace/archive/177940370e4a6b2509e92a0aaa9749184e64af43.zip
```

Comment:
There's no release?

Reply:
See this commit:

https://github.com/ianlancetaylor/libbacktrace/commit/051ecb46f84de6469f67f1731086a2961a4d2661

libbacktrace is normally an internal gcc library (meaning that it is usually built as part of gcc but it is not made available for public consumption). Its original author split it off into an independent library mainly for personal convenience, but other people started using it (most notably, I think Rust uses it as well). After some pressuring the author agreed to call a 1.0 release, but he did not actually do the github release, and I just picked the latest available git snapshot.

This library is of interest to me because it's one of the possible backends of Boost.stacktrace:

http://boostorg.github.io/stacktrace/stacktrace/configuration_and_build.html

(in particular, it provides stacktraces with file and function names, and line numbers as well)